hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java

```diff
@@ -479,6 +490,24 @@ boolean shouldStartJobs() {
                 && !partitionService.hasOnGoingMigrationLocal();
     }
 
+    private boolean allMembersHaveSameState(ClusterState clusterState) {
+        // TODO remove once the issue is fixed on the imdg side
```

Comment:
is there an issue created on HZ? if not, could we create and link to it?

Reply:
I've set the milestone to 3.12 since 3.12.1 is not created yet
https://github.com/hazelcast/hazelcast-enterprise/issues/2747